### PR TITLE
Some cleanup for 0.8 release notes

### PR DESCRIPTION
**Pull request by louiscryan**
Some cleanup for the release notes

### CODE REVIEW

File: content/about/notes/0.8.md [hunk 1]
```diff
@@ -58,5 +50,5 @@ and about [`istioctl gen-deploy`](/docs/reference/commands/istioctl/#istioctl%20
 
 - A gateway with virtual services pointing to a headless service won't work ([Issue #5005](https://github.com/istio/istio/issues/5005)).
 
-- There are [problems with Kubernetes 1.10.2](https://github.com/istio/istio/issues/5723).
+- There is a [problem with Google Kubernetes Engine 1.10.2](https://github.com/istio/istio/issues/5723).
```
Comment (linsun) [inline content/about/notes/0.8.md]: Is this only a prob with GKE?

**Comment by linsun** [comment]
@louiscryan i think you are editing on master, not 0.8 branch...

**Comment by linsun** [comment]
or should we edit on master, @geeknoid ?

I noticed the merges from @andraxylia and myself didn't show up here:  https://istio.io/about/notes/0.8/

File: content/about/notes/0.8.md [hunk 2]
```diff
@@ -29,12 +23,10 @@ Kubernetes and Cloud Foundry and works seamlessly with the routing rules.
 - **Introducing Citadel**. We've finally given a name to our security component. What was
 formerly known as Istio-Auth or Istio-CA is now called Citadel.
 
-- **Multicluster Support**. We support per-cluster Citadel in multicluster deployments such that all Citadels share the same root cert
-and workloads can authenticate each other across the mesh.
+- **Multi-Cluster Support**. We support per-cluster Citadel in multi-cluster deployments such that all Citadels share the same root certificate and workloads can authenticate each other across the mesh.
```
[the remark this answers is not in the document]
Comment (gyliu513) [reply content/about/notes/0.8.md]: Yes, when I go through the release note, I was also surprised it was only under security section. How about add a new section named as `Environments` and add multi cluster support there?

File: content/about/notes/0.8.md (hunk 1 above)
Comment (louiscryan) [reply content/about/notes/0.8.md]: yes, its a problem with the Container Optimizes OS on GKE only

**Comment by louiscryan** [comment]
I can recreate the PR on the 0.8 branch, not a big deal

@geeknoid I presume thats where we want them

**Comment by geeknoid** [comment]
It's fine to check this into master. I've been merging stuff from 0.8 to master and the other way to keep both in sync.

File: content/about/notes/0.8.md (hunk 1 above)
Comment (linsun) [reply content/about/notes/0.8.md]: ok great, thanks for clarifying it!

**Comment by geeknoid** [comment]
Replaced by #1441

**Pull request closed without merging**